scripts/ci/run-vagrant-tests.sh

```diff
@@ -16,7 +16,11 @@ cd ${GOPATH}/src/github.com/skydive-project/skydive
 
 if [ "$DEVMODE" == "true" ]; then
     make static
-    make rpm
+    make srpm
+    VERSION=$(make -s version | cut -d '-' -f 1)
+    TAG=$(make -s version | cut -d '-' -f 2- | tr '-' '.')
+    mock -r epel-7-x86_64 -D "fullver $(make -s version)" --resultdir rpmbuild/RPMS --rebuild rpmbuild/SRPMS/skydive-${VERSION}*${TAG}.src.rpm
```

Comment:
use centos-7-x86_64 target instead of epel-7  (default package build dependency)

Reply:
the full fix is
"""
    # mock centos/rhel package build as dependency are not the same (policycoreutils-python{,-utils})
    make srpm
    VERSION=$(make -s version | awk -F '-' '{print $1}')
    SHA1=$(make -s version | awk -F '-' '{print $2}')
    mock -r centos-7-x86_64 -D "fullver $(make -s version)" --rebuild rpmbuild/SRPMS/skydive-${VERSION}-*${SHA1}.src.rpm
    mkdir -p rpmbuild/RPMS/{x86_64,noarch}
    mv /var/lib/mock/centos-7-x86_64/result/skydive*-${VERSION}-*${SHA1}*.x86_64.rpm rpmbuild/RPMS/x86_64/
    mv /var/lib/mock/centos-7-x86_64/result/skydive*-${VERSION}-*${SHA1}*.noarch.rpm rpmbuild/RPMS/noarch/
"""

Reply:
Was the centos-7-x86_64 manually copied on the CI ?

Reply:
SHA1=$(make -s version | cut -d '-' -f 2- | tr '-' '.')
to handle the tainted case